packages/push/src/instruction/exec/mod.rs

```diff
@@ -85,10 +86,11 @@ mod tests {
     #[test]
     fn noop_is_correct() {
```

Comment:
Should this test be in the noop.rs file?

Reply:
Of course – that's what I get for rushing!

Reply:
(I really should be grading, but this is more fun! 😜)